.github/workflows/linkcheck.yml

```
@@ -1,37 +1,54 @@
 name: Lychee check
 on:
-  pull_request:
-    paths:
-      - '**.md'
+  repository_dispatch:
+  workflow_dispatch:
+  schedule:
+    - cron: "1 0 1 * *" # 12:01 on the 1st of each month
```

```suggestion
    - cron: "1 0 2 * *" # 12:01 on the 1st of each month
```
Temporarily set this to the 2nd so we can test it